flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-kafka/src/main/java/org/apache/flink/cdc/connectors/kafka/json/ChangeLogJsonFormatFactory.java

```diff
@@ -49,7 +49,10 @@ public class ChangeLogJsonFormatFactory {
      * @return The configured instance of {@link SerializationSchema}.
      */
     public static SerializationSchema<Event> createSerializationSchema(
-            ReadableConfig formatOptions, JsonSerializationType type, ZoneId zoneId) {
+            ReadableConfig formatOptions,
+            JsonSerializationType type,
+            ZoneId zoneId,
+            boolean isIncludedDebeziumSchema) {
```

Comment:
I intend not to modify the parameter of this method as we can extract the `isIncludedDebeziumSchema` from formatOptions.